R/piamPackages.R

```diff
@@ -0,0 +1,12 @@
+# copied from lucode2 to prevent dependency
+piamPackages <- function() {
+  packagesUrl <- "https://pik-piam.r-universe.dev/src/contrib/PACKAGES"
+  return(tryCatch({
+    sort(sub("^Package: ", "", grep("^Package: ", readLines(packagesUrl), value = TRUE)))
+  }, warning = function(w) piamPackagesStatic))
+}
+
+# this runs only when building the package
```

Comment:
Are you sure that this is run at build time? It looks more to me like this should run every time the package is being loaded.

Reply:
Yeah I was also unsure about that, but I tested and it does not run on load. My understanding now is that at build time all scripts are sourced into a namespace, and afterwards that namespace is just saved. On package load this namespace is then attached to the search path.